app/assets/stylesheets/finder_frontend.scss

```diff
@@ -379,23 +379,34 @@
 
 .facet-toggle {
   display: none;
+  margin-bottom: $gutter;
 }
 
 .js-enabled {
   .facet-toggle {
     display: inline-block;
-    margin-bottom: $gutter;
+  }
 
-    @include media(tablet) {
+  .facet-toggle__content {
+    display: block;
+
+    &.facet-toggle__content--hide {
       display: none;
     }
   }
 
-  .facet-toggle__content--hidden {
-    display: none;
+  .facet-toggle--only-on-mobile {
+    .facet-toggle {
+      @include media(tablet) {
+        display: none;
+      }
+    }
 
-    @include media(tablet) {
-      display: block;
+    .facet-toggle__content.facet-toggle__content--hide {
+      @include media(tablet) {
+        display: block;
+      }
```

Comment:
Do we need so much nesting? its making it really hard to read and understand what each class is doing. Perhaps we should group the media queries inside the class names so we don't have a lot of the same class name dotted around the file.

We could also use govuk design systems media query mixins (https://design-system.service.gov.uk/get-started/updating-your-code/#media-queries)